src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -12,6 +12,11 @@
     short-summary: Manage Azure Network resources.
 """
 
+helps['network list-usages'] = """
+    type: command
+    short-summary: List the number of network resources you are using against your subscription quota within a region.
```

Comment:
Avoid direct 2nd person language in documentation for the CLI (we also avoid it in online docs as much as possible.) For example, this documentation could read:

```
List the number of network resources used against a subscription quota within a region.
```